numberformat/diff.emu

```diff
@@ -1013,6 +1012,62 @@
     </emu-clause>
 
     <ins class="block">
+    <emu-clause id="sec-runtime-semantics-stringintlmv" type="sdo">
+      <h1>Runtime Semantics: StringIntlMV</h1>
+      <dl class="header">
+      </dl>
+      <emu-note>
+        <p>The conversion of a |StringNumericLiteral| to a Number value is similar overall to the determination of the NumericValue of a |NumericLiteral| (see <emu-xref href="#sec-literals-numeric-literals"></emu-xref>), but some of the details are different.</p>
+      </emu-note>
+      <emu-grammar>StringNumericLiteral ::: StrWhiteSpace?</emu-grammar>
+      <emu-alg>
+        1. Return *+0*.
+      </emu-alg>
+      <emu-grammar>StringNumericLiteral ::: StrWhiteSpace? StrNumericLiteral StrWhiteSpace?</emu-grammar>
+      <emu-alg>
+        1. Return StringIntlMV of |StrNumericLiteral|.
+      </emu-alg>
+      <emu-grammar>StrNumericLiteral ::: NonDecimalIntegerLiteral</emu-grammar>
+      <emu-alg>
+        1. Return 𝔽(MV of |NonDecimalIntegerLiteral|).
+      </emu-alg>
+      <emu-grammar>StrDecimalLiteral ::: `-` StrUnsignedDecimalLiteral</emu-grammar>
+      <emu-alg>
+        1. Let _a_ be StringIntlMV of |StrUnsignedDecimalLiteral|.
+        1. If _a_ is *+0*, return *-0*.
```

Comment:
I believe this should now be

```suggestion
        1. If _a_ is 0, return ~negative-zero~.
```
per https://github.com/tc39/proposal-intl-numberformat-v3/pull/78.